docs/backport_binary.md

```diff
@@ -0,0 +1,32 @@
+# backport binary
+Backporting binaries enables you to pin certain binaries during the
+provisioning process. This means you can setup your CoreOS cluster with
+specific versions of the `docker`, `fleet`, or `etcd` binary. This is how it
+looks like to pin fleet to the version currently provided by GiantSwarm. Simply
+set the following flags to your `yochu` executable.
+
+```
+--http-endpoint=https://downloads.giantswarm.io --fleet-version=v0.11.5-gs-grpc-1 --steps=distribution,overlay,fleet
+```
+
+The `--http-endpoint` defaults to `https://downloads.giantswarm.io`. We are
+using it here for completeness. This can be any other location though. The
+expected locations for `fleet` version `v0.11.5-gs-grpc-1` would be the
+following.
+
+- `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetd`
+- `https://downloads.giantswarm.io/fleet/v0.11.5-gs-grpc-1/fleetctl`
+
+See:
+ * https://github.com/giantswarm/yochu/blob/master/cli/setup_cmd.go#L36
+ * https://github.com/giantswarm/yochu/blob/master/steps/fleet/fleet.go#L34
+
+Since naming is hard here a short explanation of the steps.
+- Step `distribution` step prepares the readonly file system on CoreOS by
```

Comment:
See https://github.com/giantswarm/yochu/pull/29#discussion-diff-71420289

Let's say this step prepares the distribution path where the files will be downloaded to.